static/app/views/performance/transactionSummary/transactionReplays/index.tsx

```diff
@@ -68,11 +69,16 @@ function ReplaysContentWrapper({
   organization,
   setError,
 }: ChildProps) {
+  const eventsDisplayFilterName = decodeEventsDisplayFilterFromLocation(location);
+  const spanOperationBreakdownFilter = decodeFilterFromLocation(location);
+
   const {eventView, replays, pageLinks, isFetching, fetchError} =
     useReplaysFromTransaction({
       eventsWithReplaysView,
       location,
       organization,
+      eventsDisplayFilterName,
+      spanOperationBreakdownFilter,
     });
```

Comment:
It is nicer that all the logic stuff is shoved into the hook, and doesn't clutter up the render component. Having it all in one spot is better 👍